xbmc/input/joysticks/generic/FeatureHandling.cpp

```diff
@@ -246,7 +246,8 @@ void CAxisFeature::ProcessMotions(void)
   const bool bWasActivated = (m_state != 0.0f);
 
   if (!bActivated && bWasActivated)
-    CLog::Log(LOGDEBUG, "Feature [ %s ] on %s deactivated", m_name.c_str());
+    CLog::Log(LOGDEBUG, "Feature [ %s ] on %s deactivated", m_name.c_str(),
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)